src/inference/src/cnn_network_ngraph_impl.cpp

```diff
@@ -441,6 +442,7 @@ void CNNNetworkNGraphImpl::reshape(const std::map<std::string, ngraph::PartialSh
                 manager.register_pass<::ngraph::pass::ConvertMulticlassNmsToMulticlassNmsIE>();
                 manager.register_pass<::ngraph::pass::ConvertMatrixNmsToMatrixNmsIE>();
                 manager.register_pass<::ngraph::pass::DisableConvertConstantFoldingOnConstPath>();
+                manager.register_pass<::ov::pass::DisableDecompressionConvertConstantFolding>();
```

Comment:
I though for API 1.0 we should allow folding of compression `Convert` nodes, shouldn't we? Please clarify this